editor/components/post-taxonomies/flat-term-selector.js

```diff
@@ -172,7 +172,7 @@ class FlatTermSelector extends Component {
 		);
 		const termAddedLabel = sprintf( _x( '%s added', 'term' ), singularName );
 		const termRemovedLabel = sprintf( _x( '%s removed', 'term' ), singularName );
-		const removeTermLabel = sprintf( _x( 'Remove %s: %%s', 'term' ), singularName );
+		const removeTermLabel = sprintf( _x( 'Remove %s', 'term' ), singularName );
```

Comment:
Simplifying this logic is a good step. But now to remove tag x we say "Remove tag: x". With this change, we will say "Remove tag" for all tags should we not say the tag being removed, e.g: "Remove x"?

Reply:
"Remove tag" will be only visual. Screen reader users have the additional info provided by `aria-describedby` so they will hear, for example: `Remove tag {brief pause here} video (1 of 2)`.

Re: speech recognition software, all the buttons have now the accessible name "Remove tag" so when voicing a command like "Click Remove tag", the software will get all the buttons and offer a choice to users (Dragon NaturallySpeaking overlays numbers on UI controls with the same accessible name so users can just voice "choose 1" or "choose 2" etc.)